src/routes/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+import Login from 'screens/Login';
+import SignUp from 'screens/SignUp';
+
+function Routes() {
+  return (
+    <Router>
+      <Switch>
+        <Route path="/sign_up" component={SignUp} />
```

Comment:
You can create an object Routes and use here

```suggestion
        <Route path="ROUTES.signUp" component={SignUp} />
```
```
export const ROUTES = {
  signUp: '/sign_up',
  login: '/'
};
```